data/translations/ja.ts

```diff
@@ -69,5 +69,49 @@
         <source>Select your user and enter password</source>
         <translation>ユーザ名を選び、パスワードを入力してください</translation>
     </message>
+    <message>
+        <source>Please enter a password!</source>
+        <translation>パスワードを入力してください!</translation>
+    </message>
+    <message>
+        <source>Change password</source>
+        <translation>パスワードを変更</translation>
+    </message>
+    <message>
+        <source>Enter your username</source>
+        <translation>ユーザ名を入力</translation>
```

Comment:
While we, Japan KDE Users Group, have not renamed our organization name from 日本 KDE **ユーザ**会 to **ユーザー会** in Japanese yet, the recent trend is **ユーザー** (with 長音符: "ー") because expression without 長音符 (ー) is (or was) only used in software and some other specific industries.

```suggestion
        <translation>ユーザー名を入力</translation>
```